jupyterhub/user.py

```diff
@@ -132,6 +134,20 @@ def __init__(self, orm_user, settings, **kwargs):
             config=self.settings.get('config'),
         )
 
+    @property
+    def get_spawners(self):
+        return self._instances
+    
+    @property
+    def get_spawner(self, server_name):
```

Comment:
property methods don't take arguments. Maybe this is meant to be a regular method? It doesn't appear to be used anywhere.